packages/monaco-editor-wrapper/test/languageClientWrapper.test.ts

```diff
@@ -96,7 +96,7 @@ describe('Test LanguageClientWrapper', () => {
         const languageClientConfig: LanguageClientConfig = {
             options: {
                 $type: 'WorkerConfig',
-                url: new URL('http://localhost:63315'),
+                url: new URL('http://localhost:5173'),
```

Comment:
Arbitrary port change?

Reply:
You can specify a server port for vite, but not for vitest or it is not taken into account. This is somewhat stupid. An the latest minor version change updated the default port. Without this change the test fail in the pipeline. I am not happy with it, but currently there is no other way.